src/Security/Security/Cmdlets/Alerts/GetAlerts.cs

```diff
@@ -12,6 +12,7 @@
 // limitations under the License.
 // ------------------------------------
 
+using System;
```

Comment:
You need to add an entry to the Az.Security Changelog for this change

Reply:
Done